File: challenges/3-area-trapezio/js/naygo/areaTrapezio.js
// Input/Args: baseMaior: 8, baseMenor: 3, altura: 5
// Output: A área do trapézio é: 27,5

function areaTrapezio(baseMaior, baseMenor, altura) {

    try {
        if (isNaN((baseMaior || baseMenor || altura)))
            throw 'você não inseriu apenas números'

        return console.log((baseMaior + baseMenor) * altura / 2)

    } catch (err) {
        return console.log('Erro: ', err)
    }
}

areaTrapezio(8, 3, 5)
areaTrapezio('a', 3, 5)

File: challenges/3-area-trapezio/java/tarcila414/AreaTrapezio.java

public class AreaTrapezio {

  /**
   * Funcao para calcular a area de um trapezio
   * 
   * @param baseMenor
   * @param baseMaior
   * @param altura
   * @return
   */
  public static double calcularArea(double baseMenor, double baseMaior, double altura) {
    return (baseMenor + baseMaior) * altura / 2;
  }

  public static void main(String[] args) {
    try {
      if (args != null && args.length >= 3) {
        double baseMaior = Double.parseDouble(args[0]);
        double baseMenor = Double.parseDouble(args[1]);
        double altura = Double.parseDouble(args[2]);

        double area = calcularArea(baseMenor, baseMaior, altura);

        System.out.printf("A área do trapézio é: %.2f \n", area);
      } else {
        throw new NumberFormatException();
      }

    } catch (NumberFormatException e) {
      System.out.println("ERRO: Você deve inserir 3 valores no formato xx.xx ou x");
    }
  }
}